client/boot/index.js

```diff
@@ -165,14 +166,7 @@ function boot() {
 function renderLayout( reduxStore ) {
 	const Layout = require( 'controller' ).ReduxWrappedLayout;
 
-	const layoutElement = React.createElement( Layout, {
-		store: reduxStore
-	} );
-
-	ReactDom.render(
-		layoutElement,
-		document.getElementById( 'wpcom' )
-	);
+	renderWithReduxStore( React.createElement( Layout, { store: reduxStore } ), 'wpcom', reduxStore );
```

Comment:
![screen shot 2017-03-29 at 13 51 23](https://cloud.githubusercontent.com/assets/699132/24453145/f2e841aa-1486-11e7-932e-155dcf0d871e.png)

It looke like it should be removed from `boot`. For SSR pages we don't have this kind of issue:

![screen shot 2017-03-29 at 13 53 20](https://cloud.githubusercontent.com/assets/699132/24453173/19d162ba-1487-11e7-872f-c3232a504ec8.png)